lib/facade_converter.ts

```diff
@@ -88,6 +88,36 @@ export class FacadeConverter extends base.TranspilerBase {
     return handler && !handler(pa);
   }
 
+  buildImports(sourceFile: ts.SourceFile) {
+    var imports: {[type: string]: boolean} = {};
+
+    function getNodes(sourceFile: ts.SourceFile): ts.Node[] {
+      var nodes: ts.Node[] = [];
+      function allNodes(n: ts.Node){ts.forEachChild(n, n => {
+        nodes.push(n);
+        allNodes(n);
+        return false;
+      })};
```

Comment:
I think you're missing a `;` behind the `)`, that's why you get the awkward formatting here.